src/files-manager.ts

```diff
@@ -297,7 +297,10 @@ export class FileManager {
         temp = []
         console.info("Requesting tags to be replaced...")
         for (let file of this.ownFiles) {
-            temp.push(file.getClearTags())
+             let rem = file.getClearTags()
+             if(rem.params.notes.length) {
+                  temp.push(rem))
```

Comment:
There is a extra ')' at the end of the line, does this fix really work ?

Reply:
I don't know how it got there. It seems i committed nonsense. I am sorry, I'm going to fix it when I am at home.